perf/impl/js-libp2p/v0.45/Makefile

```diff
@@ -0,0 +1,22 @@
+commitSha := f9b91cbed9a9834ddb06fa473b76e745db90a87a
+
+all: perf
+
+perf: build-project
+	cp -r ./js-libp2p-${commitSha}/packages/perf perf-tests
+
+build-project: extract
+	docker run --rm -v "$(shell pwd)/js-libp2p-${commitSha}":/usr/src/myapp -w /usr/src/myapp node:18.17.0 npm install
+	docker run --rm -v "$(shell pwd)/js-libp2p-${commitSha}":/usr/src/myapp -w /usr/src/myapp node:18.17.0 npm run build
+
+extract: js-libp2p-${commitSha}.zip
```

Comment:
Why use a `PHONY` target even though this target actually creates files, namely the extracted `js-libp2p-${commitSha}` (note the missing `.zip`) folder?

Reply:
Good point